kotlin/openId4VP/src/main/java/io/mosip/openID4VP/networkManager/exception/NetworkManagerClientExceptions.kt

```diff
@@ -0,0 +1,9 @@
+package io.mosip.openID4VP.networkManager.exception
+
+sealed class NetworkManagerClientExceptions {
```

Comment:
why can't we have one exception for this library?
why module wise exception?

Reply:
There are many error types so in order to avoid confusion and find them easily we have added errors in module wise exception folders swati.